recipes/pybdei/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pybdei" %}
+{% set version = "0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pybdei-{{ version }}.tar.gz
+  sha256: e42082d4c16aa829e2857cb5b5760f3f12f735d288e302843980f49992907111
+
+build:
+  script_env:
+    - CC=$CXX
+  entry_points:
+    - bdei_infer = pybdei.inference:main
+    - bdei_loglikelihood = pybdei.loglikelihood:main
+    - bdei_u = pybdei.u_calculator:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --no-binary=pybdei
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pybdei', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('gxx') }}  # [linux]
```

Comment:
```suggestion
    - {{ compiler('cxx') }}
```

Reply:
Thank you @martin-g ! I've been struggling with this a lot